Dockerfile

```diff
@@ -1,57 +1,5 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
 MAINTAINER Tulio Ruiz <tulio@vauxoo.com>
 
-RUN locale-gen fr_FR \
-    && locale-gen en_US.UTF-8 \
-    && dpkg-reconfigure locales \
-    && update-locale LANG=en_US.UTF-8 \
-    && update-locale LC_ALL=en_US.UTF-8 \
-    && ln -s /usr/share/i18n/SUPPORTED /var/lib/locales/supported.d/all \
-    && locale-gen \
-    && echo 'LANG="en_US.UTF-8"' > /etc/default/locale
-
-ENV PYTHONIOENCODING="utf-8" \
-    LANG="C.UTF-8" \
-    TERM="xterm"
-
-RUN apt-get update -q && apt-get install -qy wget \
-    && wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add - \
-    && echo 'deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main' > /etc/apt/sources.list.d/pgdg.list \
```

Comment:
We dont need to install postgresql from upstream if we keep our BASE image pointing to the most recent stable Ubuntu release. As of writing, the versions are up to date.

Upstream repo:
![captura de pantalla de 2016-04-14 16-05-51](https://cloud.githubusercontent.com/assets/324683/14543159/95366556-025c-11e6-9216-c75766a6f746.png)

Ubuntu repo:
![captura de pantalla de 2016-04-14 16-07-20](https://cloud.githubusercontent.com/assets/324683/14543167/9d7ef868-025c-11e6-84d1-21ba3e964e28.png)